### PR TITLE
Profile card

### PR DESCRIPTION
**Pull request by eishikahoque**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -41,6 +41,9 @@
       content='The recent graduates of the Sheridan Interactive Media Management (IMM) are eager to share with you an immersive digital experience showcasing their design and development work. Browse through the variety of enriched work, ranging from visual design to interaction design, with a lot of coding in between. This digital showcase is to appreciate the hard work of the IMM students and to commemorate an exciting journey of endless learning and growth.' />
     <meta property='og:url' content="http://imm.sheridanc.on.ca/openhouse/2020/linkedin-og.png" />
 
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css">
```
[the remark this answers is not in the document]
Comment (eileenmsantiago) [reply public/index.html:44]: We're using Bootstrap reboot

File: src/components/ProfileModal.vue [hunk 2]
```diff
@@ -170,55 +164,56 @@ export default {
...
 
-  .fas.fa-circle {
-    font-size: 3px;
-    margin: 4px;
-  }
-
   .fas.fa-times {
-    font-size: 1.6rem;
+    font-size: 2.5rem;
   }
```
[the remark this answers is not in the document]
Comment (eishikahoque) [reply src/components/ProfileModal.vue]: Okie fixed this!

File: public/index.html [hunk 3]
```diff
@@ -41,6 +41,9 @@
...
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/milligram/1.3.0/milligram.css">
```
[the remark this answers is not in the document]
Comment (eishikahoque) [reply public/index.html]: Fixed this!

File: public/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (eishikahoque) [reply public/index.html]: Fixed this!

**Pull request merged**